packages/devtools/web/styles.css

```diff
@@ -858,6 +823,12 @@ div.tabnav-tab {
     height: 30px;
 }
 
+.form-select[disabled], .form-select[disabled]:hover {
+    color: rgba(36,41,46,0.4);
+    background-color: #eff3f6;
```

Comment:
should the colors be dependent on dark or light?

Reply:
for dark mode we just use the disabled button style for the disabled form select. If you look at styles_dark.css, I tacked `.form-select[disabled]` onto that style.

Reply:
For the `removed var` commit,  I accidentally left an unused var in my initial commit, so it was just a cleanup.

Reply:
Thanks makes sense.